static/js/common/upload-addon.js

```diff
@@ -352,6 +353,13 @@
                             $('#manual-review').show().removeClass('hidden');
                           }
                         }
+                      } else {  // This is a listed add-on.
```

Comment:
This also needs to be behing the `automaticValidation` flag. We don't want to sign beta add-ons before the automatic validation is enabled.